source/Plugins/Platform/MacOSX/PlatformDarwin.cpp

```diff
@@ -1945,3 +1946,79 @@ PlatformDarwin::LaunchProcess(lldb_private::ProcessLaunchInfo &launch_info) {
   // Let our parent class do the real launching.
   return PlatformPOSIX::LaunchProcess(launch_info);
 }
+
+lldb_private::Error
+PlatformDarwin::FindBundleBinaryInExecSearchPaths (const ModuleSpec &module_spec, Process *process, 
```

Comment:
This is largely copied from `PlatformRemoteDarwinDevice.cpp`.  It looks like we're just making sure `PlatformMacOSX` has access to it too; is that correct?  If so, I'm not going to review this logic closely.